jobs/integr8ly/pds-general.yaml

```diff
@@ -0,0 +1,300 @@
+                  stage('Execute "After Installation" Workarounds') {
+                      if (TO_DO.contains('heavy') || TO_DO.contains('install')) {
+                          if (INSTALLATION_BRANCH == 'master') {
+                              // this patches the webapp to use master image (v2.2.0 is used otherwise)
+                              sh """
+                                  oc login https://master.${YOURCITY}.openshiftworkshop.com -u admin@example.com -p Password1  --insecure-skip-tls-verify
+                                  oc patch deploymentconfig/tutorial-web-app -n webapp -p '{ "spec": { "template": { "spec": { "containers": [{ "name": "tutorial-web-app", "image": "quay.io/integreatly/tutorial-web-app:master" }]}}}}' || true
+                              """
+                          }
+                      }
+                  } // stage
+       
+                  stage('Smoke Tests') {
+                      if(TO_DO.contains('heavy') || TO_DO.contains('tests')) {
+                          buildStatus = build(job: 'installation-smoke-tests', propagate: false, parameters: [
+                              string(name: 'REPOSITORY', value: "${TEST_SUITES_REPOSITORY}"),
+                              string(name: 'BRANCH', value: "${TEST_SUITES_BRANCH}"),
+                              string(name: 'CLUSTER_URL', value: "https://master.${YOURCITY}.openshiftworkshop.com"),
+                              string(name: 'ADMIN_USERNAME', value: 'admin@example.com'),
+                              string(name: 'ADMIN_PASSWORD', value: 'Password1')]).result
+                              
+                          println "Build finished with ${buildStatus}"
+                          
+                          if(buildStatus != 'SUCCESS') {
+                              currentBuild.result = 'UNSTABLE'
+                          }
+                      } else {
+                          println 'Smoke Tests skipped'
+                      }
+                  } // stage
+
+                  stage('Browser Based Tests') {
+                      if(TO_DO.contains('heavy') || TO_DO.contains('tests')) {
+                          buildStatus = build(job: 'browser-based-testsuite-pipeline', propagate: false, parameters: [
+                              string(name: 'REPOSITORY', value: "${TEST_SUITES_REPOSITORY}"),
+                              string(name: 'BRANCH', value: "${TEST_SUITES_BRANCH}"),
+                              string(name: 'WEBAPP_URL', value: "https://tutorial-web-app-webapp.apps.${YOURCITY}.openshiftworkshop.com"),
+                              string(name: 'CLUSTER_URL', value: "https://master.${YOURCITY}.openshiftworkshop.com"),
+                              string(name: 'ADMIN_USERNAME', value: 'admin@example.com'),
+                              string(name: 'ADMIN_PASSWORD', value: 'Password1'),
+                              string(name: 'EVALS_USERNAME', value: 'eval23@example.com')]).result
```

Comment:
I think there's typo in user name. It should `evals23@example,com`

Reply:
Great catch, will fix.